src/main/java/io/dockstore/provision/S3CmdPlugin.java

```diff
@@ -67,12 +67,23 @@ public void stop() {
 
         private static final String CLIENT_LOCATION = "client";
         private static final String CONFIG_FILE_LOCATION = "config-file-location";
+        private static final String VERBOSITY = "verbosity";
         private static final String DEFAULT_CLIENT = "/usr/bin/s3cmd";
         private static final String DEFAULT_CONFIGURATION = System.getProperty("user.home") + "/.s3cfg";
+        private static final String DEFAULT_VERBOSITY = "3";
```

Comment:
Hmm, using numbers for verbosity isn't very descriptive. 
Please switch to something like DEBUG, INFO, ERROR ala logging or by just repeating `--verbose`, `-v` ala https://www.gnu.org/software/tar/manual/html_node/verbose-tutorial.html

Four levels seems a bit excessive